### ISSUE TITLE
Position size calculator

### ISSUE THREAD

**Issue by kingdomofcoders** [BigBlueBit/BitMEXAssistant#41]
![unknown](https://user-images.githubusercontent.com/39202196/43357859-d62de6a0-927f-11e8-996c-162b247595c6.png)
Populate the entry stop loss
 and target price, whether it be long or short and it will output your risk and reward and effectively produce your position size " created a Excel position size calculator as below"

My idea is you could have something similar included in your bot, and then you could have an auto fill that would populate the requested fields in the bot
More than happy to speak to in detail about this

**Comment by kingdomofcoders** [comment]
[Advanced Position Size Calculator.xlsx](https://github.com/BigBitsYouTube/BitMEXAssistant/files/2244887/Advanced.Position.Size.Calculator.xlsx)

**Comment by AJolly** [comment]
Yeah. This would be great, add options for multiple take profit levels, and let you place all those orders with one click.
